docs/plugins/algod.md

```diff
@@ -1,16 +1,17 @@
 # Algod Importer
 
-Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to
-provide old blocks.
+Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to provide old blocks.
+
+`TODO: we can't use a follower ?`
```

Comment:
^^^

Reply:
This is out of date. The plugin docs need some work: https://github.com/algorand/conduit/issues/28